virtme/mkinitramfs.py

```diff
@@ -14,6 +14,32 @@
 import itertools
 from . import cpiowriter
 from . import util
+from elftools.elf.elffile import ELFFile
+from elftools.elf.dynamic import DynamicSection
+
+ld_paths = ['/lib', '/lib65', '/usr/lib', '/usr/lib64']
+
+def is_dt_needed(tag):
+    return tag.entry.d_tag == 'DT_NEEDED'
+
+def find_library_path(needed):
+    for ld_path in ld_paths:
+        ld_path = f'{ld_path}/{needed}'
```

Comment:
If for no other reason than consistency with the rest of the code it might be slightly preferable to use `os.path.join()` here.

Reply:
Done.